### DISCUSSION TITLE
Lesson 3: Can't find 'owner' button in contract after deployment

### DISCUSSION THREAD

**Discussion by EdenSwack** [smartcontractkit/full-blockchain-solidity-course-py#1734] [Q&A] [answered]
After defining :

```
address public owner;
    
    constructor() public{
        owner = msg.sender;
    }

```
The button on remix that returns the owner address of the contract doesn't appear. I also get the following warning:

Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
  --> contracts/FundMe.sol:11:5:
   |
11 |     constructor() public{
<img width="1280" alt="Screen Shot 2022-10-04 at 23 26 41" src="https://user-images.githubusercontent.com/93931254/193920050-a96ee047-09bf-44b9-9824-db0ddbec88ef.png">

   |     ^ (Relevant source part starts here and spans across multiple lines).

**Comment by cromewar** [comment, marked as answer]
Hello @EdenSwack 
On the newest Solidity version you can skip using `public` on the constructor as it'll be public by default.
Regarding to the owner not appearing, remember first delete the latest deploy and deploy it again, still not working?

**Comment by EdenSwack** [comment]
Yes it solved the issue. Thanks!
